### PR TITLE
add back in rpm -V for java rpms since now on 7.6 rhel

### PR DESCRIPTION
**Pull request by gabemontero**
See https://jira.coreos.com/browse/ART-391

/assign @bparees 
/assign @sosiouxme

### CODE REVIEW

File: slave-base/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ RUN yum install -y centos-release-scl-rh && \
     yum install -y --setopt=tsflags=nodocs install $INSTALL_PKGS $x86_EXTRA_RPMS && \
```
Comment (sosiouxme) [inline slave-base/Dockerfile]: This still has the
the "install" stutter in it

:smile_cat:

File: slave-base/Dockerfile.rhel7 [hunk 2]
```diff
@@ -19,9 +19,7 @@ RUN yum-config-manager --disable epel >/dev/null || : && \
     x86_EXTRA_RPMS=$(if [ "$(uname -m)" == "x86_64" ]; then echo -n java-1.8.0-openjdk-headless.i686 ; fi) && \
     INSTALL_PKGS="bc gettext git java-1.8.0-openjdk-headless lsof rsync tar unzip which zip bzip2" && \
     yum install -y --setopt=tsflags=nodocs install $INSTALL_PKGS $x86_EXTRA_RPMS && \
```
Comment (sosiouxme) [inline slave-base/Dockerfile.rhel7]: this too

**Comment by gabemontero** [comment]
install stutter removed

**Comment by bparees** [comment]
/lgtm

probably won't get through testing, but i'd at least like to see that the dockerfile can get successfully built.

**Comment by gabemontero** [comment]
`rpm -V` does not like wildcards like `rh-maven33*` though `yum` tolerates them:  `package rh-maven33* is not installed`

**Comment by gabemontero** [comment]
the `rpm -V` on the javas are working again

**Comment by bparees** [comment]
> rpm -V does not like wildcards like rh-maven33* though yum tolerates them: package rh-maven33* is not installed

ok, scrap it as needed.

**Comment by bparees** [comment]
> assuming unit tests passes I say we green button @bparees

yeah that was my plan.

**Pull request merged**